### PR TITLE
Date pickers seems to work and improved to work with LocalDate/LocalD…

### PR DESCRIPTION
**Pull request by solomax**
…ateTime

### CODE REVIEW

**Comment by solomax** [comment]
Would appreciate if you can review the code :)
It seems to work now
and also seems to be more customizable :)

File: bootstrap-extensions/src/main/java/de/agilecoders/wicket/extensions/markup/html/bootstrap/form/datetime/AbstractDateTimePickerWithIcon.java [hunk 1]
```diff
@@ -116,4 +118,9 @@ protected Component newIcon(String wicketId) {
     protected IconType newIconType() {
         return FontAwesomeIconType.calendar;
     }
+
+    @Override
+    protected IMarkupSourcingStrategy newMarkupSourcingStrategy() {
```
Comment (martin-g) [inline bootstrap-extensions/src/main/java/de/agilecoders/wicket/extensions/markup/html/bootstrap/form/datetime/AbstractDateTimePickerWithIcon.java]: Maybe you can extend from `FormComponentPanel` instead ?!

File: bootstrap-samples/pom.xml [hunk 2]
```diff
@@ -51,6 +51,12 @@
             <groupId>de.agilecoders.wicket</groupId>
             <artifactId>wicket-bootstrap-themes</artifactId>
         </dependency>
+        <!-- icons -->
+        <dependency>
```
Comment (martin-g) [inline bootstrap-samples/pom.xml]: This dependency comes as transitive one from here: https://github.com/l0rdn1kk0n/wicket-bootstrap/blob/c95661ed224909f28342e45ec11f1c71c5b19241/bootstrap-extensions/pom.xml#L61-L65
Just update it from 5.7.1 to 5.9.0 there

**Review by martin-g — CHANGES_REQUESTED** [review]

**Comment by solomax** [comment]
Done :)

**Comment by martin-g** [comment]
Thank you, @solomax !

**Comment by martin-g** [comment]
I've ported the improvements to `wicket-9.x`. I haven't tested them. I hope everything still works fine.

**Pull request merged**